docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/30-common-configurations.md

```diff
@@ -0,0 +1,423 @@
+
+```yaml
+api:
+  ingress:
+    tls:
+      selfSignedCert: false
+```
+
+When setting `selfSignedCert` to `false`, the `Ingress` resource expects a
+[TLS `Secret`](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)
+with the name `kargo-api-ingress-cert` to exist in the same namespace as the
+API server.
+
+## Git Configuration
+
+Kargo supports a number of Git-related configurations that can be set at
+installation time.
+
+### Default Commit Author
+
+To set the default commit author Kargo will use when committing changes using
+the [`git-commit` Promotion step](../../50-user-guide/60-reference-docs/30-promotion-steps/git-commit.md),
+the following configuration can be set (shown here with the default values):
+
+```yaml
+controller:
+  gitClient:
+    name: Kargo
+    email: no-reply@kargo.io
+```
+
+### Signing Commits
+
+To sign commits made by Kargo, a reference to a `Secret` (in the same namespace
+as Kargo is installed to) containing a signing key can be configured. The key
+should be a GPG key in ASCII-armored format  without a passphrase, under the
+key `signingKey`.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kargo-git-signing-key
+type: Opaque
+data:
+  signingKey: <base64-encoded-ascii-armored-gpg-key>
+```
+
+The `Secret` can then be referenced in the Kargo configuration.
+
+```yaml
+controller:
+  gitClient:
+    signingKeySecret:
+      name: kargo-git-signing-key
+      type: gpg
+```
+
+:::note
+When using a signing key, the `gitClient.name` and `gitClient.email`
+configuration options must match the name and email associated with the GPG
+key.
+:::
+
+## Argo CD Configuration
+
+Kargo supports a number of Argo CD-related configurations that can be set at
+installation time.
+
+### Disabling the Argo CD Integration
+
+By default, Kargo will enable the Argo CD integration, which configures Kargo
+to work with `Application` resources created by Argo CD. This can be disabled
+as follows:
+
+```yaml
+controller:
+  argocd:
+    integrationEnabled: false
+```
+
+When not enabled, the controller will not watch Argo CD `Application` resources
+and disable Argo CD specific features. Explicitly disabling is preferred if this
+integration is not desired, as it will grant fewer permissions to the controller.
+
+### Argo CD Namespace
+
+By default, Kargo expects Argo CD to be installed to the `argocd` namespace,
+which is also the default namespace it will use for `Application` resources
+if a namespace is not specified in the
+[`argocd-update` Promotion step](../../50-user-guide/60-reference-docs/30-promotion-steps/argocd-update.md).
+
+If you want to use a different default namespace, this can be configured as
+follows:
+
+```yaml
+controller:
+  argocd:
+    namespace: argocd
+```
+
+In certain cases, you may not want Kargo to be able to look up Argo CD resources
+in other namespaces than the configured default. This can be enforced by setting
+the following configuration:
```

Comment:
Ha. I forgot that the default for this was false. It's an _opt-in_. Thanks for refreshing my memory. 😄